### DISCUSSION TITLE
Allow RGB code for theme colors.

### DISCUSSION THREAD

**Discussion by VelliusG** [BornToBeRoot/NETworkManager#1435] [Ideas]
I have web console links that shows Kibana dashboards but their dark theme is hardcoded as well... would like to be able to match the Network Manager theme to the same "dark" color.

First implementation could be done via settings.xml ... <Appearance_Theme>#1A1B20</Appearance_Theme>

**Comment by BornToBeRoot** [comment]
I could try to support custom themes (again - because it was removed by migrating mahapps.metro from 1.x to 2.x)

https://mahapps.com/docs/themes/thememanager
https://github.com/punker76/code-samples/tree/main/MahAppsMetroThemesSample

**Comment by BornToBeRoot** [comment]
Custom themes are implement via MahApps. See docs https://borntoberoot.net/NETworkManager/docs/settings/appearance#use-custom-themes
